sonic_data_client/client_test.go

```diff
@@ -402,9 +402,11 @@ func ReceiveFromZmq(consumer swsscommon.ZmqConsumerStateTable) (bool) {
 		if receivedData.Size() == 0 {
 			retry++
 			if retry >= 10 {
+				swsscommon.DeleteKeyOpFieldsValuesQueue(receivedData)
```

Comment:
We don't need to delete before return.
We can use "defer swsscommon.DeleteKeyOpFieldsValuesQueue(receivedData)" after NewKeyOpFieldsValuesQueue.

Reply:
Fixed.